### PR TITLE
feat(release): implement release with image build

### PR DESCRIPTION
**Pull request by ygrishajev**

### CODE REVIEW

File: .github/workflows/deploy-app-to-akash.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          version="${version#v}"
+          
+          echo "::set-output name=workspace::$workspace"
+          echo "::set-output name=app::$app"
+          echo "::set-output name=version::$version"
+          echo "::set-output name=image::$image"
+
+  deploy-akash:
+    name: Deploy to akash
+    uses: ygrishajev/akash-deploy-action/.github/workflows/deploy.yml@main
```
Comment (baktun14) [inline .github/workflows/deploy-app-to-akash.yml]: Interesting, perhaps we can create a fork under the akash-network org

File: .github/workflows/release-all-apps.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build the Docker images
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.email "team@akash.network"
```
Comment (baktun14) [inline .github/workflows/release-all-apps.yml]: Is this a valid email?

File: .github/workflows/release-app.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        description: Which app to release
+        options:
+          - api
+          - deploy-web
+        required: true
+      release-type:
+        type: choice
+        description: Which app to release
+        options:
+          - stable
```
Comment (baktun14) [inline .github/workflows/release-app.yml]: prod?

File: apps/api/mvm.lock [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (baktun14) [inline apps/api/mvm.lock:1]: What's this file for?

File: apps/api/package.json [hunk 5]
```diff
@@ -95,7 +96,10 @@
   },
   "devDependencies": {
     "@akashnetwork/dev-config": "*",
+    "@akashnetwork/docker": "*",
+    "@akashnetwork/releaser": "*",
     "@faker-js/faker": "^8.4.1",
+    "@release-it/conventional-changelog": "github:ygrishajev/conventional-changelog#feature/pre-release",
```
Comment (baktun14) [inline apps/api/package.json]: Perhaps we can fork this one as well in the akash-network org

File: package.json [hunk 6]
```diff
@@ -34,8 +34,11 @@
     "@akashnetwork/dev-config": "*",
     "@commitlint/cli": "^19.5.0",
     "@commitlint/config-conventional": "^19.5.0",
+    "@release-it/conventional-changelog": "github:ygrishajev/conventional-changelog#feature/pre-release",
     "cross-env": "^7.0.3",
     "husky": "^9.1.6",
+    "mvm": "^0.2.0",
```
Comment (baktun14) [inline package.json]: Minecraft version mananger is 11 years old and not maintained, is this a package you've used in the past?

Comment (baktun14) [reply package.json]: Is it this repo? https://www.npmjs.com/package/mvm https://www.npmjs.com/package/mvm

**Review by baktun14 — COMMENTED** [review]
Very nice good job!

File: .github/workflows/deploy-app-to-akash.yml (hunk 1 above)
Comment (ygrishajev) [reply .github/workflows/deploy-app-to-akash.yml]: Yea, I can do that. I was hoping for a fast reply to pr from the owner

File: .github/workflows/release-all-apps.yml (hunk 2 above)
Comment (ygrishajev) [reply .github/workflows/release-all-apps.yml]: No, do we have a valid one to serve the purpose?

File: .github/workflows/release-app.yml (hunk 3 above)
Comment (ygrishajev) [reply .github/workflows/release-app.yml]: Yea, I don't mind this var name, doesn't matter much. It's just it's more version relate rather then env. Alpha -> beta -> stable

File: apps/api/mvm.lock (hunk 4 above)
Comment (ygrishajev) [reply apps/api/mvm.lock:1]: This file is generated by that mvm package. It's keeping current monorepo package versions in the client apps. File is updated on pre-commit hook. This is needed for released to properly detect changes and bump a version

File: apps/api/package.json (hunk 5 above)
Comment (ygrishajev) [reply apps/api/package.json]: Yea, sure. Same thing, I was hoping for fast reply to my pr

File: package.json (hunk 6 above)
Comment (ygrishajev) [reply package.json]: Wow, actually found it on some article, and didn't bother much. I'll recheck the source code. It's a pretty simple package so maybe it's fine.

Comment (ygrishajev) [reply package.json]: Ok, I messed up. This is the package https://github.com/b12k/mvm

I'll remove the one commented

File: .github/workflows/release-all-apps.yml (hunk 2 above)
Comment (baktun14) [reply .github/workflows/release-all-apps.yml]: I'm not sure, but I suppose we can setup this one? cc @anilmurty

Comment (anilmurty) [reply .github/workflows/release-all-apps.yml]: we have hosting@ovrclk.com - we can use that or have Cory make a separate one for this. Up to you guys.

File: .github/workflows/release-app.yml [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build the Docker images
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.email "team@akash.network"
```
Comment (ygrishajev) [inline .github/workflows/release-app.yml]: let's ask to create developer@akash.network

**Review by baktun14 — APPROVED** [review]

**Pull request merged**